source/version.js

```diff
@@ -55,3 +55,24 @@ module.exports.validate = version => {
 		throw new Error('Version should be a valid semver version.');
 	}
 };
+
+module.exports.satisfiesRequiredVersion = async function (pkgName, installedVersion) {
+	const range = requirements[pkgName];
+	if (!satisfies(installedVersion, range)) {
+		throw new Error(`Please upgrade ${pkgName} to ${range} or higher.`);
+	}
+};
+
+/**
+ * Check whether the given version satisfies the range
+ * @private
+ * @param {string} version to check
+ * @param {string} range to check
+ * @returns {boolean} true if satisfies
+ */
+function satisfies(version, range) {
```

Comment:
I still can't understand why you've moved this to an external method instead of keeping it in the `Version` class and using it that way: `version(versionNumber).satisfies(range)`.